accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,122 @@
+# Accessibility in PDFs
+
+- [A warning](#a-warning)
+- [Is PDF the right choice?](#is-pdf-the-right-choice)
+- [Creating accessible PDF documents](#creating-accessible-pdf-documents)
+- [Creation from a word processor (e.g. Microsoft Word or Google Docs with Grackle)](#creation-from-a-word-processor-eg-microsoft-word-or-google-docs-with-grackle)
+  - [Headings](#headings)
+  - [Alt text](#alt-text)
+  - [Links](#links)
+  - [Contrast](#contrast)
+  - [Colour](#colour)
+  - [Built-in structure and navigation](#built-in-structure-and-navigation)
+  - [Tables](#tables)
+  - [Rich media](#rich-media)
+  - [Forms](#forms)
+  - [Font Size](#font-size)
+- [Further resources](#further-resources)
+- [How to fix non-accessible PDFs?](#how-to-fix-non-accessible-pdfs)
+- [References](#references)
+
+## A warning
+
+If you’re concerned about the accessibility of your Portable Document Format (PDF) documents, then you should [consult a specialist like AbilityNet](https://www.abilitynet.org.uk/creating-accessible-pdfs) to review what you have, make expert recommendations, and potentially remediate issues for you. 
```

Comment:
I think the full name 'Portable Document Format' isn't really necessary?
It's already been previously referred to as PDF and I'd say that's the only name they're known by.

Reply:
Yeah, I agree with this. We do say in the house style that we want authors to expand acronyms the first time they're used, but I agree that there are times where the acronym is better understood than the expansion (e.g. BBC or DNA), and PDF is one of them. 

What I think you could do instead is change Line 35, which currently says:

>PDF is a destination format, created from a source document...

to 

>PDF (Portable Document Format) is a destination format, created from a source document...

which makes the relationship between the acronym and its explanation specific without distracting readers during the intro.

Reply:
Thanks all